README.md

```diff
@@ -70,10 +70,10 @@ Alternatively, some manual work is required - pick one:
 
 ## Configuration
 
-Default configuration file is `/etc/pihole-updatelists.conf`.
+Default configuration file is `/etc/pihole-updatelists/pihole-updatelists.conf`.
```

Comment:
We cannot change default location of this file now as it will break the script for everyone who updates.
Instead maybe this path could be only used in docker installation (remember `--config=` parameter exists), it could also be fixed by using a symlink I believe?